libraries/AP_HAL/utility/RingBuffer.h

```diff
@@ -146,14 +146,14 @@ class ObjectBuffer {
     }
     
     /*
-      throw away an object
+      throw away an object from the front of the queue
```

Comment:
Splitting this into a separate PR would be good.

Reply:
Fixed